<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Userdashboard</class>
 <widget class="QDialog" name="Userdashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D8F8E8;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_jp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>17</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #00000
</string>
    </property>
    <property name="text">
     <string>Select Your Job Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frm_cmp">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>143</y>
     <width>216</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D8F8E8;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="img_cmp">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/all/img/cmp.png);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cmp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #0000;

</string>
    </property>
    <property name="text">
     <string>IT / CS Engineer</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frm_civil">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>143</y>
     <width>216</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D8F8E8;
border-radius: 20px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="img_civil">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/all/img/civil.png);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_civil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #00000;
</string>
    </property>
    <property name="text">
     <string>Civil Engineer</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frm_extc">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>425</y>
     <width>216</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D8F8E8;
border-radius: 20px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="img_extc">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/all/img/extc.png);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_extc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
</string>
    </property>
    <property name="text">
     <string>EXTC Engineer</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frm_mech">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>425</y>
     <width>216</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D8F8E8;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="img_mech">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/all/img/mech.png);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mech">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #0000;
</string>
    </property>
    <property name="text">
     <string>Mechanical Engineer</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#D8F8E8;</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qrc/userdash.qrc"/>
 </resources>
 <connections/>
</ui>
